apps/patient/src/graphql/queries.ts

```diff
@@ -128,53 +181,38 @@ export const GET_PHARMACIES = gql`
       is24hr: $is24hr
       name: $name
     ) {
-      id
-      name
-      address {
-        street1
-        street2
-        city
-        state
-        country
-        postalCode
-      }
-      distance
-      isOpen(at: $openAt)
-      nextEvents(at: $openAt) {
-        open {
-          ... on PharmacyOpenEvent {
-            type
-            datetime
-          }
-          ... on PharmacyCloseEvent {
-            type
-            datetime
-          }
-          ... on PharmacyOpen24HrEvent {
-            type
-          }
-        }
-        close {
-          ... on PharmacyOpenEvent {
-            type
-            datetime
-          }
-          ... on PharmacyCloseEvent {
-            type
-            datetime
-          }
-          ... on PharmacyOpen24HrEvent {
-            type
-          }
-        }
-      }
-      hours {
-        dayOfWeek
-        is24Hr
-        openFrom
-        openUntil
-        timezone
+      ...PharmacyFields
+    }
+  }
+  ${PHARMACY_FIELDS}
+`;
+
+export const GET_PHARMACIES_WITH_PRICE = gql`
+  query GetPharmaciesWithPriceByLocation(
+    $location: LatLongSearch!
+    $openAt: DateTime
+    $is24hr: Boolean
+  ) {
+    pharmaciesWithPriceByLocation(location: $location, openAt: $openAt, is24hr: $is24hr) {
+      pharmacy {
+        ...PharmacyFields
       }
+      price
+    }
+  }
+  ${PHARMACY_FIELDS}
+`;
+
+// TODO THIS IS TEMPORARY, I don't know what the query looks like yet
+export const GET_DISCOUNT_CARD = gql`
+  query GetDiscountCard($id: ID!) {
+    discountCard: order(id: $id) {
+      id
+      price: id
```

Comment:
@marisbest2 It looks like `id` got added to all of these

Reply:
Yeah, this is just a stopgap